src/elements/content-explorer/ItemGrid.js

```diff
@@ -1,19 +1,39 @@
 // @flow
 import * as React from 'react';
-import getProp from 'lodash/get';
+import ArrowKeyStepper from '@box/react-virtualized/dist/es/ArrowKeyStepper';
 import AutoSizer from '@box/react-virtualized/dist/es/AutoSizer';
+import getProp from 'lodash/get';
+
 import GridView from '../../components/grid-view/GridView';
+import { focus } from '../../utils/dom';
 import ItemGridCell from './ItemGridCell';
+
 import type { ItemGridProps } from './flowTypes';
 import type { BoxItem, Collection } from '../../common/types/core';
 
 type Props = {
     currentCollection: Collection,
     gridColumnCount: number,
+    rootElement: HTMLElement,
+    selected?: BoxItem,
```

Comment:
do these fit in `ItemGridProps`?

Reply:
doesn't look like they can, actually, because `ItemGridCell` uses `ItemGridProps`, but doesn't use these two, so the linters throw an error